lms/djangoapps/branding/migrations/0001_initial.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
```

Comment:
In general, I'm hesitant to add our own migrations to an existing djangoapp. This isn't an issue yet, but could be if edX ever adds their own model here.

This is how we run into issues w/ migrations and need to workaround with `--delete-ghost-migrations`, `--skip-ghost-migrations`, and `--merge` because DirectAccess piggy-backs on an existing model.

Before Jason left, he and I actually drafted a hypothetical plan for moving DirectAccess _out_ of `StudentProfile` and into its own model.

Is there a way for us to compartmentalize this? I'm not black-and-white on this, so let me know your thoughts.